packages/custom-elements/tests/safari-gc-bug-workaround.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+// Some distributions of Safari 10 and 11 contain a bug where non-native
+// properties added to certain globals very early in the lifetime of the page
+// are not considered reachable and might be garbage collected. This happens
+// randomly and appears to affect not only the `customElements` global but also
+// the wrapper functions it attaches to other globals, like `Node.prototype`,
+// effectively removing the polyfill. To work around this, this function checks
+// if the polyfill is missing before running the tests. If so, it uses a
+// special global function added by the polyfill during tests (which doesn't
+// get collected) to reinstall it.
+//
+// https://bugs.webkit.org/show_bug.cgi?id=172575
+export function safariGCBugWorkaround() {
```

Comment:
Let's consider just making this the global and perhaps call it `ensureCustomElementsPolyfillInstalled`.

Users who encounter this issue would otherwise have to write this function.

Reply:
After discussion, we elected not to do this since this isn't really a feature we want to encourage using. If the native feature is available, users should be using it.